monitoring/osd-configuring-the-monitoring-stack.adoc

```diff
@@ -0,0 +1,115 @@
+
+* See xref:../monitoring/osd-configuring-the-monitoring-stack.adoc#preparing-to-configure-the-monitoring-stack[Preparing to configure the monitoring stack] for steps to create monitoring config maps
+* xref:../monitoring/osd-enabling-monitoring-for-user-defined-projects.adoc#enabling-monitoring-for-user-defined-projects[Enabling monitoring for user-defined projects]
+* See the link:https://docs.openshift.com/container-platform/4.6/nodes/scheduling/nodes-scheduler-taints-tolerations.html#nodes-scheduler-taints-tolerations[OpenShift Container Platform documentation] on taints and tolerations
+* See the link:https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/[Kubernetes documentation] on taints and tolerations
+
+// Configuring persistent storage
+[id="configuring-persistent-storage"]
+== Configuring persistent storage
+
+Running cluster monitoring with persistent storage means that your metrics are stored to a persistent volume (PV) and can survive a pod being restarted or recreated. This is ideal if you require your metrics or alerting data to be guarded from data loss. For production environments, it is highly recommended to configure persistent storage. Because of the high IO demands, it is advantageous to use local storage.
+
+[IMPORTANT]
+====
+See link:https://docs.openshift.com/container-platform/4.6/scalability_and_performance/optimizing-storage.html#recommended-configurable-storage-technology_persistent-storage[Recommended configurable storage technology].
+====
+
+[id="persistent-storage-prerequisites"]
+=== Persistent storage prerequisites
+
+* Dedicate sufficient local persistent storage to ensure that the disk does not become full. How much storage you need depends on the number of pods. For information on system requirements for persistent storage, see xref:../scalability_and_performance/scaling-cluster-monitoring-operator.adoc#prometheus-database-storage-requirements[Prometheus database storage requirements].
+
+* Make sure you have a persistent volume (PV) ready to be claimed by the persistent volume claim (PVC), one PV for each replica. Because Prometheus has two replicas and Alertmanager has three replicas, you need five PVs to support the entire monitoring stack. The PVs should be available from the Local Storage Operator. This does not apply if you enable dynamically provisioned storage.
+
+* Use the block type of storage.
+
+* link:https://docs.openshift.com/container-platform/4.6/storage/persistent_storage/persistent-storage-local.html[Configure local persistent storage.]
+
+include::modules/osd-monitoring-configuring-a-local-persistent-volume-claim.adoc[leveloffset=+2]
+include::modules/osd-monitoring-modifying-retention-time-for-prometheus-metrics-data.adoc[leveloffset=+2]
+
+.Additional resources
+
+* See xref:../monitoring/osd-configuring-the-monitoring-stack.adoc#preparing-to-configure-the-monitoring-stack[Preparing to configure the monitoring stack] for steps to create monitoring config maps
```

Comment:
The "Preparing to configure the monitoring stack" subsection does not exist in this ROSA book. Is this because the steps to create the config map is not relevant to this platform? You could either remove or change this cross-reference to reflect this.

Reply:
removed.